Etabs_Adapter/CRUD/Read/Results/MeshResults.cs

```diff
@@ -508,14 +698,48 @@ private List<MeshStress> SmoothenStresses(List<MeshStress> forces)
                 double pr2 = group.Average(x => x.Principal_2);
                 double pr1_2 = group.Average(x => x.Principal_1_2);
 
-                smoothenedForces.Add(new MeshStress(first.ObjectId, first.NodeId, "", first.ResultCase, first.ModeNumber, first.TimeStep, first.MeshResultLayer, first.LayerPosition, MeshResultSmoothingType.ByPanel, first.Orientation, sxx, syy, sxy, txx, tyy, pr1, pr2, pr1_2));
+                smoothenedForces.Add(new MeshStress(first.ObjectId, first.NodeId, "", first.ResultCase, first.ModeNumber, first.TimeStep, 
+                                                        first.MeshResultLayer, first.LayerPosition, MeshResultSmoothingType.ByPanel, first.Orientation, 
+                                                        sxx, syy, sxy, txx, tyy, pr1, pr2, pr1_2));
```

Comment:
Why have these been modified? This is going to confuse people who look back on this in a few years wondering what changes you made to `MeshForce` either via the PR or Blame.

Reply:
@peterjamesnugent, looks to me here just the indentation of the code has changed. Would you prefer me to turn code lines alignment to their original?

Reply:
@peterjamesnugent, code lines restored as per they original setting out ;)